activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/test/java/org/activiti/cloud/services/modeling/service/ModelServiceImplTest.java

```diff
@@ -305,6 +315,18 @@ public void should_throwException_when_validatingAnInvalidJSONModelContentFileIn
         assertThatThrownBy(() -> modelService.validateModelContent(modelOne, fileContent, projectOne)).isInstanceOf(SemanticModelValidationException.class);
     }
 
+    @Test
+    public void should_returnErrors_when_gettingValidationErrorOfAnInvalidModel() throws Exception {
+        when(modelOne.getType()).thenReturn(modelType.getName());
+
+        when(modelContentService.findModelValidators(modelType.getName())).thenReturn(List.of(modelContentValidator));
+        doThrow(new SemanticModelValidationException(List.of(new ModelValidationError())))
+            .when(modelContentValidator)
+            .validateModelContent(any(), any());
+
+        assertThat(modelService.getModelValidationErrors(modelOne, ValidationContext.EMPTY_CONTEXT)).isNotEmpty();
```

Comment:
Maybe a more specific assertion can be used here, instead of just checking if it's not empty.

Reply:
Done.